src/platform/packages/shared/response-ops/rules-apis/query_keys.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const queryKeys = {
+  root: 'rules',
+  getRuleTags: ({
```

Comment:
Trying out the WIP proposal of using the same exact name as the fetchFn for the key creator (for better discoverability)